app/templates/header.html

```diff
@@ -38,9 +36,18 @@
     </div>
 </header>
 
-
+<!-- Macro call removed for translation syntax to function correctly -->
 <div class="govuk-width-container">
-    {{ govukPhaseBanner({'tag': {'text': 'beta'}, 'html':
-    'This is a new service – your <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=EGg0v32c3kOociSi7zmVqN48ORk8WN5LlJITE3Swt-lURUNCR0dHMjgxWFZOMTMxQzlOTVIxVkQ0Sy4u" class="govuk-link">feedback</a> will help us to improve it.'
-    })}}
```

Comment:
@gidsg Ive tried to add gettext to this and babel does not capture the strings. Can you confirm that i should use the html instead of nunjuxks?

Reply:
fixed, no need to go over this